locales/en-US/server.ftl

```diff
@@ -17,6 +17,9 @@ screenshotsLogo =
     .title = Screenshots Home
 bannerSignIn = <a>Sign in or sign up</a> to access your shots across devices and save your favorites forever.
 bannerUpsell = {gScreenshotsDescription} <a>Get Firefox now</a>
+# These two messages are presented in a banner on all pages, like: <b>shutdownWarningIntro</b> shutdownWarningBody
+shutdownWarningIntro = Saved screenshots are expiring soon.
```

Comment:
Is there a reason for this not being a single string? Technically it should work.

```
shutdownWarningIntro = 
    <b>Saved screenshots are expiring soon.</b> Starting in June, Screenshots
    will no longer offer online storage. Want to keep shots from your library?
    <a>Download them to your computer.</a>
```